packages/shared/src/lib/featureManagement.ts

```diff
@@ -47,4 +48,8 @@ export const featureOnboardingSources = new Feature(
   false,
 );
 
+export const featureFeedAdTemplate = new Feature('feed_ad_template', {
+  adStart: 2,
```

Comment:
Doesn't this need default position as well here? (to mimic current pagesize)

Reply:
Since it depends on pageSize I added a default fallback inside useFeed when adRepeat is `undefined`

Reply:
See https://github.com/dailydotdev/apps/pull/3894/files#diff-e2dd034a87d4027cdddb729a7e7ed2fbd456558adcc0eaeee1364b40dba26417R199

Reply:
Once `adRepeat` comes from GB page size does not matter anymore